native/src/routes/CityNotCooperating.tsx

```diff
@@ -80,8 +79,10 @@ const TemplateText = styled.Text`
 
 const CityNotCooperating = (): ReactElement => {
   const { t } = useTranslation('cityNotCooperating')
+  const { cityNotCooperatingIcon } = buildConfigAssets()
   const [isCopied, setIsCopied] = useState<boolean>(false)
   const theme = useTheme()
+  const template: string = buildConfig().featureFlags.cityNotCooperatingTemplate!
 
   const copyToClipboard = () => {
     Clipboard.setString(template)
```

Comment:
```suggestion
    if (template) {
      Clipboard.setString(template)
      setIsCopied(true)
    }
```
i think its safer to check here and avoid the not null assertion above

Reply:
only allowed the route when the template is set and there keep the not null assertion above.